db/migrate/20210816100932_fix_setting_category.rb

```diff
@@ -0,0 +1,5 @@
+class FixSettingCategory < ActiveRecord::Migration[6.0]
```

Comment:
I've just hitted an issue of name clash, because in ansible https://github.com/theforeman/foreman_ansible/pull/448 we took example from this PR and Rails complain about the same name.
It would be probably better to play it safe and name it more specific (both the class and the file needs to be renamed).